Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM fedora:29 as BUILDENV
+RUN dnf --assumeyes install redhat-rpm-config
+RUN dnf --assumeyes install rubygem-bundler
+RUN curl -sL https://dl.yarnpkg.com/rpm/yarn.repo -o /etc/yum.repos.d/yarn.repo
+RUN dnf --assumeyes install yarn
+RUN dnf --assumeyes install git
+RUN dnf --assumeyes group install "C Development Tools and Libraries"
+RUN dnf --assumeyes install ruby-devel
+RUN dnf --assumeyes install libffi-devel
+USER nobody
+WORKDIR /tmp
+RUN git clone https://github.com/ashphy/jsonpath-online-evaluator.git
```

Comment:
Please prevent to use git cloning for the tracking the head commit in the docker hub automated build.